app/change-password/page.tsx

```diff
@@ -0,0 +1,94 @@
+"use client";
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { useSearchParams } from "next/navigation";
+
+const ResetPassword: React.FC = () => {
+  const [newPassword, setNewPassword] = useState<string>("");
+  const [confirmPassword, setConfirmPassword] = useState<string>("");
+  const [message, setMessage] = useState<string>("");
+  const [error, setError] = useState<string>("");
+  const searchParams = useSearchParams();
+  const token = searchParams.get("token");
+
+  //const { token } = queryParameterValue as String;
+
+  useEffect(() => {
+    if (!token) {
+      setError("Invalid or expired token.");
+    }
+  }, [token]);
+
+  const handleSubmit = async (event: React.FormEvent) => {
+    event.preventDefault();
+    setMessage("");
+    setError("");
+
+    if (newPassword !== confirmPassword) {
+      setError("Passwords do not match.");
+      return;
+    }
+
+    try {
+      // Send a request to the backend to reset the password
+      const response = await axios.post("/api/reset-password", {
+        token,
+        newPassword,
+      });
+
+      if (response.data.success) {
+        setMessage("Password has been reset successfully.");
+      } else {
+        setError("Failed to reset password. Token may have expired.");
+      }
+    } catch (err) {
+      setError("An error occurred. Please try again.");
+    }
+  };
+
+  return (
+    <div className="flex items-center justify-center min-h-screen bg-white">
+      <div className="max-w-md w-full p-6 bg-white rounded-lg">
+        <h1 className="text-2xl font-semibold mb-6 text-center">
```

Comment:
A lot of redundant CSS here as in app/recover-password/page.tsx, please consolidate them into css classes.